src/todo-list/components/AddTodoForm.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import styled from 'styled-components';
+import useTodoList from '../hooks/useTodoList';
+
+function AddTodoForm() {
+
+    const { createTodo } = useTodoList();
+  
+    const onSubmit = (event) => {
+      event.preventDefault();
+      createTodo(event.target[0].value);
+      event.target[0].value = '';
+    } 
+
+    return (
+        <StyledFooter>
+            <StyledForm class="add-todo-form" onSubmit={onSubmit}>
+                <StyledInputContainer>
+                    <StyledInput type="text" placeholder="새로운 할 일 추가" />
+                </StyledInputContainer>
+                <StyledBtn type="submit" >
+                    추가
+                </StyledBtn>
+            </StyledForm>
+        </StyledFooter>
+    )
+}
+
+const StyledFooter = styled.footer`
+  height: 10%;
+  /* width: 100%; */
```

Comment:
```suggestion
  height: 10%;
```

이 부분은 지워 주셔도 될 것 같습니다! 그나저나 엄청 깔끔하게 작성하셔서 감탄하면서 보는 중이에요... 👍